javascripts/discourse/components/home-list.gjs

```diff
@@ -79,6 +89,27 @@ export default class HomeList extends Component {
     }
   }
 
+  @action
+  scrollTop() {
+    // if someone prefers reduced motion, skip animating
+    const noMotionPreference = window.matchMedia(
+      "(prefers-reduced-motion: no-preference)"
+    ).matches;
+
+    this.rocketLaunch = true;
+    setTimeout(
```

Comment:
When dealing with ember/glimmer code (in this case: settings a `tracked` prop) you should use `discourseLater()` instead of `setTimeout`